types/tags.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	cmn "github.com/tendermint/tmlibs/common"
+)
+
+// Type synonym for convenience
+type Tag = cmn.KVPair
+
+// Type synonym for convenience
+type Tags cmn.KVPairs
+
+// New empty tags
+func EmptyTags() Tags {
+	return make(Tags, 0)
+}
+
+// Append a single tag
+func (t Tags) AppendTag(k string, v []byte) Tags {
+	return append(t, MakeTag(k, v))
+}
+
+// Append two lists of tags
+func AppendTags(a, b Tags) Tags {
+	return append(a, b...)
```

Comment:
Why not make a function of tags? `a.AppendTags(b)`

Reply:
Done.